about.html

```diff
@@ -22,13 +24,19 @@
 That includes behaviour from other instances - we have no problem with stopping to federate with asshole instances, or suspend contact with single accounts.
 </p>
 
-<hr>
+<h2>Publication rules</h2>
+
+We try to keep the <a href="https://chaos.social/web/timelines/public/local">local timeline</a> of our instance and the Fediverse in general a place of dialogue and interaction. Therefore we established some rules. Please be aware, that any repeated disregard of these rules will lead to silencing or suspension of your account.
```

Comment:
perhaps it could be added that people will get some kind of notification beforehand. Right now it sounds like a single offense will get you silenced (without any mentioning of it). And I don't believe this is what happens.

Reply:
No, you're right, that will not happen for things like crossposting. We always try about three times to reach people first. I'll try to find a way to make that clear.

Reply:
The second sentence is superflous, and the third is, as well, as we already say in the very beginning that misbehaviour can lead to silencing or suspension of accounts.

Reply:
Yes, it is said for the Code of Conduct which measures will be taken. I was not expecting that the same enforcement also applies to the **Publication rules**.

Reply:
Reformulated so the "enforcement" paragraph is at the beginning and is understandably to be applied to all rules in **General behaviour** (Code of Conduct) and **Publication rules**.

Reply:
Based on @ccoenen feedback I also added a sentence into the enforcement paragraph: *Usually we tell you a couple of times in advance, before taking any of these measures.*